lib/airbrake/rake/tasks.rb

```diff
@@ -15,7 +15,23 @@
       "#{name}:\n  " + [cfg, filters].pretty_inspect
     end.join("\n")
 
-    if response
+    if !response
+      puts <<NORESPONSE
```

Comment:
What do you think of [adding some margin based on which to trim the excessive whitespace](https://github.com/bbatsov/ruby-style-guide#heredocs)?

```ruby
puts <<-NORESPONSE.gsub(/^\s+\|/, '')
  |Couldn't send a test exception. There are two reasons for this:
  |1. Airbrake ignored this exception due to misconfigured filters
  |...
NORESPONSE